### ISSUE TITLE
Hold/ gain column triggers accessibility error

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov_elections#118]
@Mjeffbrown tells us

The hold/gain column of the results summary lacks a heading, and is being picked up as a level A accessibility fail on SiteImprove.

See it on https://new.newcastle.gov.uk/election/uk-parliamentary-general-election-4-july-2024
